aikau/src/main/resources/alfresco/forms/controls/css/MultiSelect.css

```diff
@@ -168,11 +168,11 @@
    }
    &--has-error {
       border-color: #c00;
-      outline: 1px solid #faa;
+      box-shadow: 0 0 1px 1px #ffc4c4;
```

Comment:
Good question. The replaced outline values weren't LESS variables, but perhaps they also should have been. The colours are different to account for the different rendering of the two properties, and were judged to be similar by eye. We could certainly make them LESS variables, although it'd be good to tie them (using LESS algorithms) to the colours of our global outline values.
How do you want to do it?